.github/workflows/iroha2-dev-pr.yml

```diff
@@ -119,6 +121,7 @@ jobs:
   test-docker:
     #runs-on: [ self-hosted, Linux ] #ubuntu-latest
     runs-on: ubuntu-latest
+    timeout-minutes: 20
```

Comment:
it is better to do timeouts in our code than in ci

Reply:
If we have some bugs with actors/async it runs for 6 hours :-/

Reply:
I guess you right

Reply:
Changed to 60 minutes.